lib/presentation/pages/intro_page.dart

```diff
@@ -57,10 +57,11 @@ class _IntroPageState extends State<IntroPage> {
           PageViewModel(
             titleWidget: const SizedBox(),
             decoration: emptyPageDecoration(),
-            bodyWidget: StudyTypeSelectionMolecule(() {
+            bodyWidget: StudyTypeSelectionMolecule(() async {
               setState(() {
                 studyType = StudyTypeAbstract.instance!.studyType.name;
               });
+              await Future.delayed(const Duration(milliseconds: 200));
```

Comment:
This duration should be const for all use cases in page

Reply:
I will make both these changes, I am about to open another PR for issue #18 so I will include these changes in that as well if its fine?

Reply:
External (at class level) const Duration var, not inside each transition